modules/csg/doc_classes/CSGShape3D.xml

```diff
@@ -93,7 +93,7 @@
 			Snap makes the mesh vertices snap to a given distance so that the faces of two meshes can be perfectly aligned. A lower value results in greater precision but may be harder to adjust. The top-level CSG shape's snap value is used for the entire CSG tree.
 		</member>
 		<member name="use_collision" type="bool" setter="set_use_collision" getter="is_using_collision" default="false">
-			Adds a collision shape to the physics engine for our CSG shape. This will always act like a static body. Note that the collision shape is still active even if the CSG shape itself is hidden. See also [member collision_mask] and [member collision_priority].
+			Adds a collision shape to the physics engine for our CSG shape. This node will always act like a static body with a collision shape, not a collision shape. Note that the collision shape is still active even if the CSG shape itself is hidden. See also [member collision_mask] and [member collision_priority].
```

Comment:
Error in the wording:

> with a collision shape, not a collision shape.

Reply:
I believe I made it more clear and accurate.